### DISCUSSION TITLE
What conditions are met to be invited to join the organization?

### DISCUSSION THREAD

**Discussion by fawdlstty** [ros2-rust/ros2_rust#331] [General]
Hello, I would like to join the Github organization of ROS2 RUST to display on my Github personal page. I would like to know if there are specific criteria for determining which conditions should be met to join the organization?

**Comment by esteve** [comment]
@fawdlstty there's no formal process, but basically when someone has contributed to the project on a regular basis, we (@jhdcs , @nnmm and I ) just add the person to the organization.
